classes/class-wc-connect-shipping-method.php

```diff
@@ -437,7 +437,8 @@ public function calculate_shipping( $package = array() ) {
 						'cost'      => $rate->rate,
 						'meta_data' => array(
 							'wc_connect_packages' => $rate->packages,
-							__( 'Packaging', 'woocommerce-services' ) => $packaging_info
+							__( 'Packaging', 'woocommerce-services' ) => $packaging_info,
+							'wc_connect_packing_log' => $rate->packing_log,
```

Comment:
The property is called `box_packing_log` on the server. Did you want to change it?